tests/unit/store/entity/actions.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { actions } from '@/store/entity/actions';
+import { ENTITY_INIT } from '@/store/entity/actionTypes';
+import { ENTITY_INIT as ENTITY_INIT_MUTATION } from '@/store/entity/mutationTypes';
+import { factory } from '@/common/TermboxFactory';
+import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
+import Entity from '@/store/entity/Entity';
+import EntityNotFound from '@/common/data-access/error/EntityNotFound';
+
+function newMinimalStore( fields: any ): Entity {
+	return {
+		id: 'Q1',
+		labels: {},
+		descriptions: {},
+		aliases: {},
+		...fields,
+	} as Entity;
+}
+
+describe( 'entity/actions', () => {
+	describe( ENTITY_INIT, () => {
+		it( `commits to ${ENTITY_INIT_MUTATION} on successful FingerprintableEntity lookup`, ( done ) => {
+			const entityId = 'Q42';
+
+			const entity = new FingerprintableEntity( entityId, {}, {}, {} );
+			factory.setEntityRepository( {
+				getFingerprintableEntity: ( thisEntityId: string ) => {
+					expect( thisEntityId ).toBe( entityId );
+					return Promise.resolve( entity );
+				},
+			} );
+			const context = newMinimalStore( {
+				commit: ( mutationName: string, thisEntity: FingerprintableEntity ) => {
```

Comment:
I know that we already have similar tests with the assertions in the mock callback, but I don't think they read very nicely. What do you think about
```typescript
			const commitMock = jest.fn();
			const context = newMinimalStore( {
				commit: commitMock,
			} );
			const action = actions[ ENTITY_INIT ] as any; // TODO

			action( context, entityId ).then( () => {
				expect( commitMock ).toBeCalledWith(
					ENTITY_INIT_MUTATION,
					entity,
				);
				done();
			} )
```

<sup>(I would've done this as an inline suggestion but I don't know how to make it multi line)</sup>

Reply:
Using this for `language/actions.spec.ts`